src/UI/Art/SpriteArtist/ControlOptionsSetupArtist.java

```diff
@@ -23,22 +24,33 @@
   }
 
   private static SpriteCursor spriteCursor = new SpriteCursor();
+  private static OptionSelectionContext context;
 
   public static void draw(Graphics g, InputHandler control)
   {
     // Draw a fancy background.
     DiagonalBlindsBG.draw(g);
 
     myControl = control;
+    if( null == context )
+    {
+      context = new OptionSelectionContext();
+      SpriteOptions.initialize(InputHandler.extraOptions, context);
```

Comment:
Feels like `OptionSelectionContext` should be broken out into its own class/file, and probably have its own `initialize()` call instead of having this artist call that artist to initialize this artist's stuff.
I'd also change the name of the new class to like, `OptionPanelTemplate`/`Configuration` or something, I dunno.